Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/EyelinkBasicLegacy/EyelinkDoDriftCorrect.m

```diff
@@ -0,0 +1,4 @@
+function result=EyelinkDoDriftCorrect(el, x, y, draw, allowsetup)
+    warning('EyelinkToolbox:LegacyDoDriftCorrect',['The function EyelinkDoDriftCorrect() is deprecated. Please update ', ...
```

Comment:
Actual legacy code missing!

Reply:
resolved in 33984c0ef6fc9d2a3dc40f30ab9710ad031d9938